frontend/Gemfile

```diff
@@ -1,7 +1,7 @@
 eval(File.read(File.dirname(__FILE__) + '/../common_spree_dependencies.rb'))
 
-gem 'solidus_core', path: '../core'
-gem 'solidus_api', path: '../api'
 gem 'generator_spec', group: :test
+gem 'solidus_api', path: '../api'
+gem 'solidus_core', path: '../core'
```

Comment:
I suggest disabling the Bundler/OrderedGems cop instead of doing this. Alphabetical ordering is second to group by functionality / ordering by importance, in my mind at least.